pkg/oc/admin/policy/reconcile_sccs.go

```diff
@@ -188,45 +195,44 @@ func (o *ReconcileSCCOptions) ChangedSCCs() ([]*securityapi.SecurityContextConst
 		actualSCC, err := o.SCCClient.Get(expectedSCC.Name, metav1.GetOptions{})
 		// if not found it needs to be created
 		if kapierrors.IsNotFound(err) {
-			changedSCCs = append(changedSCCs, expectedSCC)
+			toCreateSCCs = append(toCreateSCCs, expectedSCC)
 			continue
 		}
 		if err != nil {
-			return nil, err
+			return nil, nil, err
 		}
 
 		// if found then we need to diff to see if it needs updated
 		if updatedSCC, needsUpdating := o.computeUpdatedSCC(*expectedSCC, *actualSCC); needsUpdating {
-			changedSCCs = append(changedSCCs, updatedSCC)
+			toUpdateSCCs = append(toUpdateSCCs, updatedSCC)
 		}
 	}
-	return changedSCCs, nil
+	return toUpdateSCCs, toCreateSCCs, nil
 }
 
 // ReplaceChangedSCCs persists the changed SCCs.
-func (o *ReconcileSCCOptions) ReplaceChangedSCCs(changedSCCs []*securityapi.SecurityContextConstraints) error {
-	for i := range changedSCCs {
-		_, err := o.SCCClient.Get(changedSCCs[i].Name, metav1.GetOptions{})
-		if err != nil && !kapierrors.IsNotFound(err) {
-			return err
-		}
+func (o *ReconcileSCCOptions) ReplaceChangedSCCs(
+	newSCCs []*securityapi.SecurityContextConstraints,
+	changedSCCs []*securityapi.SecurityContextConstraints) error {
```

Comment:
In Go, it's possible to specify a type only once for both arguments:
```
func (o *ReconcileSCCOptions) ReplaceChangedSCCs(newSCCs, changedSCCs []*securityapi.SecurityContextConstraints) error {
```

Reply:
Thanks, I completely forgot about this feature. It'll make the whole thing much nicer :slightly_smiling_face: